packages/app/src/cli/services/function/build.ts

```diff
@@ -162,6 +162,28 @@ function getESBuildOptions(
   return esbuildOptions
 }
 
+export async function runWasmOpt(modulePath: string) {
+  const wasmOpt = wasmOptBinary()
+  await downloadBinary(wasmOpt)
+
+  const wasmOptDir = dirname(wasmOptBinary().path)
+
+  const command = `node`
+  const args = [
+    // invoke the js-wrapped wasm-opt binary
+    wasmOptBinary().name,
+    modulePath,
+    // pass these options to wasm-opt
```

Comment:
🤔 How are you thinking we'd get these non-default options?
Should it be something we can pass to `build`?
This area seems a bit of an internal concern to me, since this is how the CLI will invoke the `wasm-opt` binary by default.

If users want to do their own "custom" invocations of wasm-opt, they could use the `--disable-wasm-opt` flag I've just added, then invoke it themselves. Is that sufficient?

Reply:
Yeah good point -- will it be used? We can wait for feedback on this. They can always create their own build script if they want to try different options.

The disabling should be in the TOML though, not a CLI flag. e.g.

```
[extensions.build]
wasm_opt = false #defaults to true
```

Reply:
Done. Made it pull from the config instead. Do we need to update the docs as well to include this new option then?

Reply:
Yeah good call, it would be here:
https://shopify.dev/docs/apps/build/app-extensions/configure-app-extensions#shopify-functions-extensions

Reply:
https://github.com/Shopify/shopify-dev/pull/51556